jest.config.ts

```diff
@@ -15,5 +15,9 @@ export default async (): Promise<Config.InitialOptions> => {
    transform: { "^.+\\.tsx?$": "ts-jest" },
    globalTeardown: "<rootDir>/src/tests/jest-globals-teardown.ts",
    forceExit: true,
+   coveragePathIgnorePatterns: [
+    "/node_modules/",
+    "<rootDir>/src/tests/**/*.test.ts",
```

Comment:
line 20: this path is invalid -> got the test fail

Reply:
> line 20: this path is invalid -> got the test fail

Noted, I will check